### ISSUE TITLE
Doesn't run on Ubuntu 18.04

### ISSUE THREAD

**Issue by scottjl** [jimnarey/alu_auto_builder#1]
looks like some hard-coded requirements for glibc?

```
❯ ./ucetool
./ucetool: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /src/ucetool_linux_beta_2/lib/libpython3.8.so.1.0)
./ucetool: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /src/ucetool_linux_beta_2/lib/libpython3.8.so.1.0)
```

**Comment by jimnarey** [comment]
Thanks for this. I'm using cx_freeze to package the app and it's clearly requiring a version of glibc not available on Ubuntu 18 (which uses 2.27). I'm going to try building on Ubuntu 18 and seeing if that then works on 20.

**Comment by jimnarey** [comment]
Hi - I've released Beta 3 which I built on Ubuntu 18 and have confirmed also works with Ubuntu 20. Let me know if you have any more problems, but it should be fine.
